tests/link/pytorch/test_extra_ops.py

```diff
@@ -41,3 +41,50 @@ def test_pytorch_CumOp(axis, dtype):
         out = pt.cumprod(a, axis=axis)
         fgraph = FunctionGraph([a], [out])
         compare_pytorch_and_py(fgraph, [test_value])
+
+
+@pytest.mark.parametrize("axis", [0, 1])
+def test_pytorch_Repeat(axis):
+    a = pt.matrix("a", dtype="float64")
+
+    test_value = np.arange(6, dtype="float64").reshape((3, 2))
+
+    out = pt.repeat(a, (1, 2, 3) if axis == 0 else (3, 3), axis=axis)
+    fgraph = FunctionGraph([a], [out])
+    compare_pytorch_and_py(fgraph, [test_value])
+
+
+@pytest.mark.parametrize("axis", [0, 1])
+def test_pytorch_Unique_axis(axis):
+    a = pt.matrix("a", dtype="float64")
+
+    test_value = np.array(
+        [[1.0, 1.0, 2.0], [1.0, 1.0, 2.0], [3.0, 3.0, 0.0]], dtype="float64"
+    )
+
+    out = pt.unique(a, axis=axis)
```

Comment:
Does PyTensor only allows integer axis at the moment? No `None` or partial axis? If we do, we should test

Reply:
All good with this one?

Reply:
I didn't check the code, but we need to see if multiple (but not all) axis is supported by PyTensor and if so test.

Reply:
Do you mean multiple axis like in the case of `ArgMax` and `Max`?
I don't think so:
>     axis : int, optional